### PR TITLE
CO-1741 openapi design

### PR DESCRIPTION
**Pull request by tsoliangwu0130**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -12,86 +12,91 @@ info:
...
+          example: Harry
+        - name: filter[lastName]
+          in: query
+          description: Last name of a person. Must be used with `filter[firstName]` together.
+          required: false
+          schema:
+            type: string
+            nullable: false
+          example: Potter
+        - name: filter[username]
```
Comment (Ruefa) [inline openapi.yaml]: I feel like this should be `onid` instead of `username` which is what I did in persons. thoughts?

File: openapi.yaml [hunk 2]
```diff
@@ -280,18 +157,86 @@ components:
...
           type: array
           items:
             $ref: '#/components/schemas/ErrorObject'
+    IdentityId:
+      type: string
+      description: Unique ID of identity resource
+    IdentityType:
+      type: string
+      enum:
+        - identity
```
Comment (Ruefa) [inline openapi.yaml]: I think this should be `enum: [identity]`

File: openapi.yaml [hunk 3]
```diff
@@ -280,18 +157,86 @@ components:
...
+    IdentityAttributes:
+      type: object
+      properties:
```
Comment (Ruefa) [inline openapi.yaml:184]: Can you add examples?

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (tsoliangwu0130) [reply openapi.yaml]: Sounds good, will do.

**Review by Ruefa — APPROVED** [review]

**Pull request merged**